prow/pjutil/pjutil.go

```diff
@@ -59,26 +61,36 @@ func newProwJob(spec kube.ProwJobSpec, extraLabels, extraAnnotations map[string]
 	}
 }
 
-// NewPresubmit converts a config.Presubmit into a kube.ProwJob.
-// The kube.Refs are configured correctly per the pr, baseSHA.
-// The eventGUID becomes a github.EventGUID label.
-func NewPresubmit(pr github.PullRequest, baseSHA string, job config.Presubmit, eventGUID string) kube.ProwJob {
+func createRefs(pr github.PullRequest, baseSHA string) kube.Refs {
 	org := pr.Base.Repo.Owner.Login
 	repo := pr.Base.Repo.Name
+	repoLink := pr.Base.Repo.HTMLURL
 	number := pr.Number
-	kr := kube.Refs{
-		Org:     org,
-		Repo:    repo,
-		BaseRef: pr.Base.Ref,
-		BaseSHA: baseSHA,
+	return kube.Refs{
+		Org:      org,
+		Repo:     repo,
+		RepoLink: repoLink,
+		BaseRef:  pr.Base.Ref,
+		BaseSHA:  baseSHA,
+		BaseLink: fmt.Sprintf("%s/commit/%s", repoLink, baseSHA),
```

Comment:
Can we deserialize these from the GitHub API response and not need to construct them?

Reply:
Digging through the [Github API docs](https://developer.github.com/v3/activity/events/types/#pullrequestevent), I can't find the exact fields we want. It seems reasonable to switch the `BaseLink` for postsubmits to `compare` (which compares the `Before` and `After` commits), but I'll leave the other two for now unless I've missed something.